src/components/wishlistCard/index.js

```diff
@@ -0,0 +1,124 @@
+import {
```

Comment:
Hey, @RohankumarM  if you have made your fork branch equal with the upstream branch then the wishlistCard component is not present in  ` src/components/wishlistCard/index.js ` so please remove this file from PR :)